examples/Example5_Fit_Halo_mass_to_DES_data.ipynb

```diff
@@ -0,0 +1,821 @@
+    "\n",
+    "Name | RA (deg) | DEC (deg) | Z_LAMBDA | LAMBDA | Note\n",
+    "- | - | - | - | - | -\n",
+    "RMJ025415.5-585710.7 | 43.564574 | -58.95297 | 0.429804 | 234.50368 | --\n",
+    "RMJ051637.4-543001.6 | 79.155704 | -54.500456 | 0.30416065 | 195.06956 | ACO S520\n",
+    "RMJ224851.8-443106.3 | 342.215897 | -44.518403 | 0.3514858 | 178.83827 | --\n",
+    "...\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c684515e-e176-4b92-9509-0217ade681a0",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
+    "tags": []
+   },
+   "source": [
+    "<a id=\"Downloading_the_catalog\"></a>\n",
+    "## 3. Downloading the catalog at the cluster field\n",
+    "\n",
+    "We consider RMJ051637.4-543001.6 (ACO S520) as an example.\n",
+    "We can access the DES catalog from NOIRLab Data Lab (https://datalab.noirlab.edu/query.php?name=des_dr1.shape_metacal_riz_unblind). No registration is required.\n",
+    "We make the query and download the catalogs in \"Query Interface\". \n",
+    "We use `coadd_objects_id` to cross match the shape catalog and photo-z catalog (https://datalab.noirlab.edu/query.php?name=des_dr1.photo_z). \n",
+    "Since the cluster is at redshift about 0.3, a radius of 0.3 deg would be about a radial distance of 5 Mpc. \n",
+    "The final catalog includes shape info and photo-z. \n",
+    "Here is an example of the query SQL command. \n",
+    "The query could take a few minutes and the size of the catalog is about 1.4 MB (.csv). \n",
+    "\n",
+    "```\n",
+    "SELECT P.mean_z, \n",
+    "C.ra, C.dec, C.e1, C.e2, C.r11, C.r12, C.r21, C.r22 \n",
+    "FROM des_dr1.photo_z as P\n",
+    "INNER JOIN des_dr1.shape_metacal_riz_unblind as C\n",
+    "ON P.coadd_objects_id=C.coadd_objects_id\n",
+    "WHERE 't' = Q3C_RADIAL_QUERY(C.ra, C.dec,79.155704, -54.500456, 0.3) \n",
+    "AND P.minchi2<1\n",
+    "AND P.z_sigma<0.1\n",
+    "AND C.flags_select=0\n",
+    "\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f4535f6-23b6-46e8-b9f8-7aeb899bfe7c",
+   "metadata": {},
+   "source": [
+    "<a id=\"Loading_the_catalog\"></a>\n",
+    "## 4. Loading the catalog into CLMM\n",
+    "\n",
+    "Once we have the catalog, we read in the catalog, make cuts on the catalog, and adjust column names to prepare for the analysis in CLMM.\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 125,
+   "id": "a04d966e-e00b-4e07-94d9-c3cc13b6183c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 4.23 ms, sys: 0 ns, total: 4.23 ms\n",
+      "Wall time: 3.11 ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "# Assume the downloaded catalog is at this path:\n",
+    "filename = \"../../../ACOS520_DES.csv\"\n",
+    "catalog = filename.replace('.csv', '.pkl')\n",
```

Comment:
@shenmingfu  I'm running the notebook and the query give a `.txt` file, not a `.csv`.

Reply:
Yes, the suffix needs to be adjusted manually from .txt to .csv. The text inside is in the csv format.